### ISSUE TITLE
Rootless install out of date?

### ISSUE THREAD

**Issue by traverseda** [gobolinux/gobolinux.org#12]
https://www.gobolinux.org/?page=rootless still appears pretty prominently on google search, but my understanding is that it is no-longer possible. If it is still possible, then linking to more information about the `CreateRootlessEnvironment` would be good. Especially where it can be found.

If it isn't still possible, it would be good to mark that page as out of date, or remove it entirely.

**Comment by acook** [comment]
Seconding this. I'd say either provide a link and instructions or remove it. It'd be great if it was still possible though. :)

**Comment by lucasvr** [comment]
The last reference to Rootless has been removed from gobolinux.org now. Hopefully Google will catch up and remove it from the search results in the days to come.

**Comment by waldyrious** [comment]
> The last reference to Rootless has been removed from gobolinux.org now.

For the record: 96e32e5f7 and 6d8a2726a.

**Comment by 320kg** [comment]
https://www.gobolinux.org/?page=rootless still there

**Comment by lonestead** [comment]
> https://www.gobolinux.org/?page=rootless still there

Second this, I found it from a link in a blog linking to an archived gobolinux.org, then I switched to the modern website. I was wondering why there was no link to download the CreateRootlessEnvironment script.

Bummer if it no longer exists...I'm running NixOS, but want a setup with more simple and straightforward design philosophies, but maybe I don't need or want a rootless environment anyways, maybe I could roll my own with chroot lol.
